utils/index.js

```diff
@@ -0,0 +1,5 @@
+import config from './config'
+
+const { NETWORK_ADDRESS_PREFIX } = config
+
+export const getShortNameAddress = (address) => `${NETWORK_ADDRESS_PREFIX}:${address}`
```

Comment:
Could we reuse this new function in other files like `app_layout.test.js`, `safe_balances.test.js`...

you can search by `NETWORK_ADDRESS_PREFIX`:
<img width="1148" alt="Captura de pantalla 2021-11-23 a las 16 24 26" src="https://user-images.githubusercontent.com/26763673/143052803-ece84d80-e2e4-4c78-a3b3-6fb77b4ce5b8.png">